### PR TITLE
Update constructor pinnings

### PR DESCRIPTION
**Pull request by hmaarrfk**
Closes https://github.com/conda-forge/miniforge/issues/127
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
So i don't know why pyyaml 6.0 is still broken. Any idea @conda-forge/miniforge ?

File: build_miniforge.sh [hunk 1]
```diff
@@ -9,11 +9,28 @@
 # See also: [setup-qemu-action](https://github.com/docker/setup-qemu-action)
 set -ex
 
+HOST_ARCH=$(uname -m)
```
Comment (isuruf) [inline build_miniforge.sh:12]:
```suggestion
BUILD_ARCH=$(uname -m)
```

**Comment by ericpre** [comment]
> So i don't know why pyyaml 6.0 is still broken. Any idea @conda-forge/miniforge ?

Maybe because of https://github.com/conda-forge/miniforge/blob/5c6c2fbbefe14d3d3ff33f4421956137f9e67ac6/scripts/build.sh#L27

On windows, the constructor 3.2.2 requires a new version of conda-standalone: https://github.com/conda-forge/conda-standalone-feedstock/pull/24 because of https://github.com/conda/constructor/issues/480 - it is a long story with some egg and chicken issues.
If anyone from @conda-forge/core, @conda-forge/conda-standalone or @conda-forge/miniforge  could give an additional review to the one of @jaimergp already did, that would be great!

File: build_miniforge.sh (hunk 1 above)
Comment (hmaarrfk) [reply build_miniforge.sh]: Thank you!

**Pull request closed without merging**